nexus/api.go

```diff
@@ -22,6 +23,7 @@ const (
 	headerOperationState = "Nexus-Operation-State"
 	headerOperationID    = "Nexus-Operation-Id"
 	headerRequestID      = "Nexus-Request-Id"
+	headerLinks          = "Nexus-Links"
```

Comment:
```suggestion
	headerLink          = "Nexus-Link"
```

Reply:
It could have multiple values. Is it convention to have header names always in singular?

Reply:
I don't think there's convention around this.
I wanted to use the singular form because each header key value pair represents a single link in our use case but I'd be fine with going with plural too.

Reply:
It is fairly common for HTTP headers that accept multiple values (they can be provided comma delimited or as separate lines) to be singular. For instance it's `Accept-Language` not `Accept-Languages`.

Reply:
I'm using the same header for the HTTP response, which we know it can contain multiple links (although it's only one for our first use case).
I'll change to singular.